release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -681,10 +681,18 @@ The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated f
 [id="ocp-4-16-node-bug-fixes_{context}"]
 ==== Node
 
+* Previously, when manually restarting the `kubelet` service on a node, some state files were deleted in anticipation of a node reboot. As the node is not rebooted in this scenario, the `kubelet` service re-assigns CPU resources for running workloads. As a result, the CPU configuration after the kubelet restart might differ from the initial `cpuset` configuration. With this update, the `cpuset` configuration is correctly restored after a kubelet restart. (link:https://issues.redhat.com/browse/OCPBUGS-24366[*OCPBUGS-24366*])
+
 [discrete]
 [id="ocp-4-16-node-tuning-operator-bug-fixes_{context}"]
 ==== Node Tuning Operator (NTO)
 
+* Previously, the Performance Profile Creator (PPC) incorrectly populated the `metadata.ownerReferences.uid` field for Day 0 performance profile manifests. As a consequence, it was not possible to apply a performance profile at Day 0 without manual intervention. With this release, the PPC does not generate the `metadata.ownerReferences.uid` field for Day 0 manifests. As a result, you can apply a performance profile manifest at Day 0 as expected. (link:https://issues.redhat.com/browse/OCPBUGS-29751[*OCPBUGS-29751*])
+
+* Previously, the `tuned` and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration. As a consequence, conflicts in the IRQ CPU affinity configuration might cause unexpected behaviour after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration.(link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
+
+* Previously, when restarting a {sno} cluster with a performance profile applied, a race condition caused systemd processes to remain in the root directory instead of being moved to `/sys/fs/cgroup/cpuset/system.slice`. With this release, systemd processes are moved to `/sys/fs/cgroup/cpuset/system.slice` as expected following a {sno} node restart. (link:https://issues.redhat.com/browse/OCPBUGS-31692[*OCPBUGS-31692*])
```

Comment:
@rbaturov - can you please review this 4.16 bug fix release note? Thanks!

Reply:
Looks good to me.